### PR TITLE
feat(docs): Add Table of Contents

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Include TOC for better navigation experience.

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @aslushnikov

**Comment by jamesgeorge007** [comment]
cc @aslushnikov

**Comment by jamesgeorge007** [comment]
cc @aslushnikov

File: README.md [hunk 1]
```diff
@@ -25,6 +25,18 @@ Most things that you can do manually in the browser can be done using Puppeteer!
 
 Give it a spin: https://try-puppeteer.appspot.com/
 
+## Table of Contents
```
Comment (aslushnikov) [inline README.md]: let's use our TOC-generator - just add a  comments here instead of the actual TOC, and then run `npm run doc`

Comment (jamesgeorge007) [reply README.md]: The respective error seems to pop up ( with `npm run doc` ) even after the successful execution of `npm i` :thinking: 
```md
> puppeteer@1.18.1-post doc /home/james-the-hacker/CodeSpace/GitHub-Projects/puppeteer
> node utils/doclint/cli.js

(node:6073) UnhandledPromiseRejectionWarning: Error: Chromium revision is not downloaded. Run "npm install" or "yarn install"
    at Launcher.launch (/home/james-the-hacker/CodeSpace/GitHub-Projects/puppeteer/lib/Launcher.js:119:15)
    at <anonymous>
(node:6073) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6073) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Comment (jamesgeorge007) [reply README.md]: @aslushnikov ??

Comment (aslushnikov) [reply README.md]: @jamesgeorge007 did you try running "npm i" like the error recommends?

Comment (jamesgeorge007) [reply README.md]: Seems to be an issue with my local environment. The issue still persists :confused:

Comment (cap-errouissi) [reply README.md]: Fix

**Comment by jamesgeorge007** [comment]
cc @aslushnikov

**Comment by aslushnikov** [comment]
On a second thought, I'm hesitant to add TOC to the Readme currently - it might break our export to developers.google.com, which is very fragile.

Let's not do this!

**Pull request closed without merging**